Frontend/linkedout/src/main/pages/LoginPage/LoginPage.tsx

```diff
@@ -35,6 +35,13 @@ const LoginPage = () => {
                 >
                     Login
                 </Button> 
+                <Button 
+                    className='register-btn' 
+                    onClick={handleLoginBtnClick}
```

Comment:
Okay, so the _handleLoginBtnClick_ method handles making an API call to log the user in. This method shouldn't be here, 

Instead make a new handler for it. Name it something like _handleRegistrationRedirect_. 

Take a look at the hooks file for the LoginPage. If you look at the _handleLoginBtnClick_ method you'll see there's a variable called navigate that's used for redirection.

EDIT 2: Also remove the href prop, the navigate variable in the hooks file handles all the redirection so you don't have to!

Reply:
Thank you for these suggestions! I will implement them and notify you when it's done.